legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -552,6 +553,13 @@ def alter_outputs_special_resolution(filing, outputs):
                 outputs.remove('certifiedRules')
         return outputs
 
+    @staticmethod
+    def alter_outputs_agm_location_change(filing, outputs):
+        """Handle output file list modification for agm location change."""
+        if filing.filing_type == 'agmLocationChange':
+            outputs.add('letterOfAgmLocationChange')
+        return outputs
```

Comment:
don't think this is required.

I think how consent to continuation out filing achieves this is by adding to the additional property in ../core/meta/filing.py.  You will need to test things out though.

![image](https://github.com/bcgov/lear/assets/6685223/31866842-5067-413e-9f12-020d7f256d14)

![image](https://github.com/bcgov/lear/assets/6685223/865ea549-6d02-4e41-a81d-6770e91c1bae)

Reply:
fix it and looks good for the PDF and document list